[BUG]: Rust newb can't install 1.10.0

### Pre-submission checks

- [x] I am **not** filing a feature request. These should be filed via the feature request form instead.
- [x] I have looked through the [open issues](https://github.com/zizmorcore/zizmor/issues?q=is%3Aissue+is%3Aopen+) for a duplicate report.

### zizmor version

1.10.0

### Expected behavior

I tried to install 1.10.0 and got a Rust compilation error.  I used `brew update` and `brew upgrade` to get the latest tools, but it still fails:
```
% cargo install zizmor
    Updating crates.io index
  Installing zizmor v1.10.0
    Updating crates.io index
     Locking 355 packages to latest compatible versions
      Adding tree-sitter-bash v0.23.3 (available: v0.25.0)
   Compiling proc-macro2 v1.0.95
   Compiling unicode-ident v1.0.18
   Compiling libc v0.2.174
   Compiling serde v1.0.219
   Compiling memchr v2.7.5
   Compiling cfg-if v1.0.1
   Compiling itoa v1.0.15
   Compiling shlex v1.3.0
   Compiling once_cell v1.21.3
   Compiling pin-project-lite v0.2.16
   Compiling bytes v1.10.1
   Compiling stable_deref_trait v1.2.0
   Compiling version_check v0.9.5
   Compiling futures-core v0.3.31
   Compiling smallvec v1.15.1
   Compiling cc v1.2.27
   Compiling autocfg v1.5.0
   Compiling futures-sink v0.3.31
   Compiling pin-utils v0.1.0
   Compiling futures-io v0.3.31
   Compiling futures-channel v0.3.31
   Compiling slab v0.4.10
   Compiling futures-task v0.3.31
   Compiling fnv v1.0.7
   Compiling typenum v1.18.0
   Compiling log v0.4.27
   Compiling ryu v1.0.20
   Compiling generic-array v0.14.7
   Compiling tracing-core v0.1.34
   Compiling http v1.3.1
   Compiling thiserror v2.0.12
   Compiling utf8parse v0.2.2
   Compiling getrandom v0.3.3
   Compiling serde_json v1.0.140
   Compiling writeable v0.6.1
   Compiling hashbrown v0.15.4
   Compiling litemap v0.8.0
   Compiling equivalent v1.0.2
   Compiling bitflags v2.9.1
   Compiling quote v1.0.40
   Compiling icu_properties_data v2.0.1
   Compiling percent-encoding v2.3.1
   Compiling ring v0.17.14
   Compiling icu_normalizer_data v2.0.0
   Compiling syn v2.0.104
   Compiling socket2 v0.5.10
   Compiling mio v1.0.4
   Compiling indexmap v2.9.0
   Compiling num-traits v0.2.19
   Compiling aho-corasick v1.1.3
   Compiling anyhow v1.0.98
   Compiling regex-syntax v0.8.5
   Compiling zeroize v1.8.1
   Compiling rustls-pki-types v1.12.0
   Compiling getrandom v0.2.16
   Compiling http-body v1.0.1
   Compiling untrusted v0.9.0
   Compiling httparse v1.10.1
   Compiling base64 v0.22.1
   Compiling block-buffer v0.10.4
   Compiling crypto-common v0.1.6
   Compiling thiserror v1.0.69
   Compiling tower-service v0.3.3
   Compiling unicode-width v0.1.14
   Compiling powerfmt v0.2.0
   Compiling rustix v1.0.7
   Compiling rustls v0.23.28
   Compiling try-lock v0.2.5
   Compiling anstyle v1.0.11
   Compiling want v0.3.1
   Compiling deranged v0.4.0
   Compiling digest v0.10.7
   Compiling form_urlencoded v1.2.1
   Compiling cpufeatures v0.2.17
   Compiling errno v0.3.13
   Compiling anstyle-parse v0.2.7
   Compiling utf8_iter v1.0.4
   Compiling num-conv v0.1.0
   Compiling heck v0.5.0
   Compiling is_terminal_polyfill v1.70.1
   Compiling time-core v0.1.4
   Compiling colorchoice v1.0.4
   Compiling subtle v2.6.1
   Compiling anstyle-query v1.1.3
   Compiling anstream v0.6.19
   Compiling num-integer v0.1.46
   Compiling sync_wrapper v1.0.2
   Compiling fst v0.4.7
   Compiling time v0.3.41
   Compiling syn v1.0.109
   Compiling ipnet v2.11.0
   Compiling regex-automata v0.4.9
   Compiling tower-layer v0.3.3
   Compiling lazy_static v1.5.0
   Compiling webpki-roots v1.0.1
   Compiling lock_api v0.4.13
   Compiling tree-sitter-language v0.1.5
   Compiling parking_lot_core v0.9.11
   Compiling iri-string v0.7.8
   Compiling clap_lex v0.7.5
   Compiling ucd-trie v0.1.7
   Compiling prettyplease v0.2.35
   Compiling synstructure v0.13.2
   Compiling same-file v1.0.6
   Compiling either v1.15.0
   Compiling zerocopy v0.8.26
   Compiling ref-cast v1.0.24
   Compiling crossbeam-utils v0.8.21
   Compiling strsim v0.11.1
   Compiling clap_builder v4.5.40
   Compiling uriparse v0.6.4
   Compiling walkdir v2.5.0
   Compiling serde_derive v1.0.219
   Compiling zerofrom-derive v0.1.6
   Compiling yoke-derive v0.8.0
   Compiling zerovec-derive v0.11.1
   Compiling displaydoc v0.2.5
   Compiling futures-macro v0.3.31
   Compiling tokio-macros v2.5.0
   Compiling zerofrom v0.1.6
   Compiling thiserror-impl v2.0.12
   Compiling yoke v0.8.0
   Compiling tracing-attributes v0.1.30
   Compiling tokio v1.45.1
   Compiling futures-util v0.3.31
   Compiling zerovec v0.11.2
   Compiling zerotrie v0.2.2
   Compiling rustls-webpki v0.103.3
   Compiling tracing v0.1.41
   Compiling thiserror-impl v1.0.69
   Compiling tinystr v0.8.1
   Compiling icu_locale_core v2.0.0
   Compiling potential_utf v0.1.2
   Compiling icu_collections v2.0.0
   Compiling regex v1.11.1
   Compiling icu_provider v2.0.0
   Compiling icu_properties v2.0.1
   Compiling icu_normalizer v2.0.0
   Compiling miette-derive v5.10.0
   Compiling Inflector v0.11.4
   Compiling futures-executor v0.3.31
   Compiling idna_adapter v1.2.1
   Compiling idna v1.0.3
   Compiling serde_urlencoded v0.7.1
   Compiling url v2.5.4
   Compiling pest v2.8.1
   Compiling miette v5.10.0
   Compiling ref-cast-impl v1.0.24
   Compiling hyper v1.6.0
   Compiling tokio-rustls v0.26.2
   Compiling tower v0.5.2
   Compiling tree-sitter v0.25.6
   Compiling schemafy_core v0.6.0
   Compiling tower-http v0.6.6
   Compiling clap_derive v4.5.40
   Compiling hyper-util v0.1.14
   Compiling num-bigint v0.4.6
   Compiling sha-1 v0.10.1
   Compiling sha2 v0.10.9
   Compiling http-body-util v0.1.3
   Compiling vte_generate_state_changes v0.1.2
   Compiling ahash v0.8.12
   Compiling hyper-rustls v0.27.7
   Compiling fastrand v2.3.0
   Compiling adler2 v2.0.1
   Compiling scopeguard v1.2.0
   Compiling arrayvec v0.7.6
   Compiling rustversion v1.0.21
   Compiling object v0.36.7
   Compiling base64 v0.21.7
   Compiling xxhash-rust v0.8.15
   Compiling regex-syntax v0.6.29
   Compiling unicode-width v0.2.1
   Compiling hex v0.4.3
   Compiling portable-atomic v1.11.1
   Compiling vte v0.11.1
   Compiling ssri v9.2.0
   Compiling num-rational v0.4.2
   Compiling tempfile v3.20.0
   Compiling reqwest v0.12.20
   Compiling schemafy_lib v0.6.0
   Compiling miniz_oxide v0.8.9
   Compiling clap v4.5.40
   Compiling regex-automata v0.1.10
   Compiling pest_meta v2.8.1
   Compiling tokio-stream v0.1.17
   Compiling toml_datetime v0.6.11
   Compiling serde_spanned v0.6.9
   Compiling http-serde v2.1.1
   Compiling futures v0.3.31
   Compiling async-trait v0.1.88
   Compiling num-iter v0.1.45
   Compiling num-complex v0.4.6
   Compiling sha1 v0.10.6
   Compiling uuid v1.17.0
   Compiling memmap2 v0.5.10
   Compiling tree-sitter-yaml v0.7.1
   Compiling quick-xml v0.37.5
   Compiling csv-core v0.1.12
   Compiling reflink-copy v0.1.26
   Compiling toml_write v0.1.2
   Compiling overload v0.1.1
   Compiling borrow-or-share v0.2.2
   Compiling bit-vec v0.8.0
   Compiling gimli v0.31.1
   Compiling toml_edit v0.22.27
   Compiling fluent-uri v0.3.2
   Compiling bit-set v0.8.0
   Compiling plist v1.7.2
   Compiling nu-ansi-term v0.46.0
   Compiling csv v1.3.1
   Compiling num v0.4.3
   Compiling cacache v13.1.0
   Compiling serde-sarif v0.8.0
   Compiling crossbeam-epoch v0.9.18
   Compiling http-cache-semantics v2.1.0
   Compiling pest_generator v2.8.1
   Compiling matchers v0.1.0
   Compiling parking_lot v0.12.4
   Compiling vt100 v0.15.2
   Compiling addr2line v0.24.2
   Compiling console v0.15.11
   Compiling bincode v1.3.3
   Compiling typed-builder-macro v0.21.0
   Compiling sharded-slab v0.1.7
   Compiling tree-sitter-powershell v0.25.6
   Compiling tree-sitter-bash v0.23.3
   Compiling tracing-log v0.2.0
   Compiling bstr v1.12.0
   Compiling thread_local v1.1.9
   Compiling rustc-demangle v0.1.25
   Compiling outref v0.5.2
   Compiling streaming-iterator v0.1.9
   Compiling nohash-hasher v0.2.0
   Compiling text-size v1.1.1
   Compiling number_prefix v0.4.0
   Compiling httpdate v1.0.3
   Compiling camino v1.1.10
   Compiling unsafe-libyaml v0.2.11
   Compiling owo-colors v4.2.2
   Compiling vsimd v0.8.0
   Compiling typed-builder v0.21.0
   Compiling http-cache v0.21.0
   Compiling indicatif v0.17.11
   Compiling uuid-simd v0.8.0
   Compiling line-index v0.1.2
   Compiling serde_yaml v0.9.34+deprecated
   Compiling globset v0.4.16
   Compiling tracing-subscriber v0.3.19
   Compiling backtrace v0.3.75
   Compiling strum_macros v0.27.1
   Compiling pest_derive v2.8.1
   Compiling referencing v0.30.0
   Compiling zizmor v1.10.0
   Compiling crossbeam-deque v0.8.6
   Compiling fraction v0.15.3
   Compiling os_info v3.12.0
   Compiling toml v0.8.23
   Compiling fancy-regex v0.14.0
   Compiling reqwest-middleware v0.4.2
   Compiling clap_complete v4.5.54
   Compiling email_address v0.2.9
   Compiling itertools v0.14.0
   Compiling xattr v1.5.1
   Compiling filetime v0.2.25
   Compiling crc32fast v1.4.2
   Compiling bytecount v0.6.9
   Compiling home v0.5.11
   Compiling num-cmp v0.1.0
   Compiling strum v0.27.1
   Compiling etcetera v0.10.0
   Compiling jsonschema v0.30.0
   Compiling flate2 v1.1.2
   Compiling github-actions-expressions v0.0.7
   Compiling tar v0.4.44
   Compiling clap_complete_nushell v4.5.7
   Compiling http-cache-reqwest v0.15.2
   Compiling human-panic v2.0.2
   Compiling ignore v0.4.23
   Compiling yamlpath v0.23.0
   Compiling tracing-indicatif v0.3.9
   Compiling github-actions-models v0.31.0
   Compiling clap-verbosity-flag v3.0.3
   Compiling annotate-snippets v0.11.5
   Compiling terminal-link v0.1.0
error[E0063]: missing field `remove_opts` in initializer of `CACacheManager`
  --> /Users/ned/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/zizmor-1.10.0/src/github_api.rs:90:22
   |
90 |             manager: CACacheManager {
   |                      ^^^^^^^^^^^^^^ missing `remove_opts`

For more information about this error, try `rustc --explain E0063`.
error: could not compile `zizmor` (bin "zizmor") due to 1 previous error
error: failed to compile `zizmor v1.10.0`, intermediate artifacts can be found at `/var/folders/6j/khn0mcrj35d1k3yylpl8zl080000gn/T/cargo-installKsiN5f`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```
I'm assuming I'm missing something about Rust to make this work, but it's always worked for me before to just use `cargo install zizmor`.

### Actual behavior

See output above.

### Reproduction steps

`% cargo install zizmor` on a Mac. 

### Logs

```plain text

```

### Additional context

_No response_

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -34,7 +34,7 @@ csv = "1.3.1"
 etcetera = "0.10.0"
 flate2 = "1.1.2"
 fst = "0.4.7"
-http-cache-reqwest = "0.15.1"
+http-cache-reqwest = "0.15.2"
 human-panic = "2.0.1"
 ignore = "0.4.23"
 indexmap = { version = "2.9.0", features = ["serde"] }

--- a/crates/zizmor/src/github_api.rs
+++ b/crates/zizmor/src/github_api.rs
@@ -89,6 +89,7 @@ impl Client {
             mode: CacheMode::Default,
             manager: CACacheManager {
                 path: cache_dir.into(),
+                remove_opts: Default::default(),
             },
             options: HttpCacheOptions {
                 cache_options: Some(CacheOptions {

